spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1686,20 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+**Built-in Directives**
+
+GraphQL implementations should provide the `@skip` and `@include` directives.
+When returning the set of directives from the `__Schema` introspection type, both
+`@skip` and `@include` directives must be included.
+
+GraphQL implementations that support the type system definition language must
+provide the `@deprecated` directive if representing deprecated portions of
+the schema, in this case `@deprecated` directive may be included when returning
+the set of directives from the `__Schema` introspection type.
+
+When representing a GraphQL schema using the type system definition language,
+the built-in directives should be omitted for brevity.
```

Comment:
What about stream and defer in this context. We are printing stream and defer in our implementation since I consider them built-in but kind of optional. I know they are not yet in the spec, but on their way, so it would be great to get an understanding if we just consider these three directives built-in and thus omit-able or if all future spec directives shall be omitted as well.

Reply:
There's flexibility here, hence the "should". I think you're reading this correctly though - it's safe to omit them once they become built-in to the spec, or even earlier since they're on that path.

There are definitely cases where you might want to include them though, in case you're communicating what is and isn't possible in a service, since not all services support those two new directives.

I wonder if I we should downgrade this to [MAY](https://tools.ietf.org/html/rfc2119)

Reply:
+1 for MAY
I think in general, it's more not which are standard and which are not. @defer and @stream are challenging for implementation, way more than @deprecated, so some servers might not support it, for a while at least. So showing @stream or not in SDL and intro is more question of actual support by the server/app.